cmd/device-util/driver.go

```diff
@@ -0,0 +1,159 @@
+//go:build windows
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	cli "github.com/urfave/cli/v2"
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+const pollTick = 5 * time.Millisecond
+
+// flag names
+
+const (
+	driverNameFlag     = "name"
```

Comment:
Personally I've found it helpful to put the `flag` part of the name first, like `flagDriverName`. This makes it easier to use intellisense to discover all the flags at once.